compatibility-tests/install-ses.js

```diff
@@ -0,0 +1,14 @@
+
+// alas '-r esm' vs (native) ESM means we can't import real modules if we
+// start with '-r esm', so import the built distfile. Don't forget to rebuild
+// this file (cd packages/ses && yarn build) after any changes to the SES
+// source files.
+//import { lockdown } from '../packages/ses/src/main.js';
+import { lockdown } from '../packages/ses/dist/ses.cjs';
```

Comment:
In #315, this usage pattern changes.

Reply:
You mean #307, right?

I'll update the imports after that PR lands.

Reply:
@warner It landed

Reply:
Thanks, yes, #307 was the correct `globalThis` change!